packages/lab/src/dialog/DialogTitle.tsx

```diff
@@ -26,19 +25,19 @@ interface DialogTitleProps
   /**
    * Displays the Dialog Title in a H2 component
    */
-  title: ReactNode;
+  header: ReactNode;
   /**
    * Displays the Dialog Subtitle in a Label component
    **/
-  subtitle?: ReactNode;
+  preheader?: ReactNode;
 
   className?: string;
 }
 
 export const DialogTitle = ({
```

Comment:
I'm not against the naming but now we have a weird situation where the component is called DialogTitle and everything else is a header. It feels like it should now be called DialogHeader.

Reply:
yep agreed